.github/workflows/docker.yml

```diff
@@ -39,7 +39,20 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
         if: github.event_name != 'pull_request'
-      - name: Build
+      - name: Build and make available
+        uses: docker/build-push-action@v6
+        with:
+          tags: ${{ steps.docker_meta.outputs.tags }}
+          labels: ${{ steps.docker_meta.outputs.labels }}
+          file: Dockerfile
+          load: true
```

Comment:
I see `Build and make available` twice, before and after `Test image`

Is this expected?